test/test_distributions.py

```diff
@@ -102,12 +102,12 @@ def is_all_nan(tensor):
     ]),
     Example(Beta, [
         {
-            'concentration1': torch.tensor(torch.exp(torch.randn(2, 3)), requires_grad=True),
-            'concentration0': torch.tensor(torch.exp(torch.randn(2, 3)), requires_grad=True),
+            'concentration1': torch.randn(2, 3).exp().requires_grad_(),
```

Comment:
I think @weiyangfb 's fixing these warnings.

Reply:
That's right, my previous PR #11061 incorrectly raises warnings at `internal_new_from_data` whenever input is a Tensor. This was fixed by https://github.com/pytorch/pytorch/pull/11815 and only raise warnings at `torch.tensor()` and `tensor.new_tensor()`

Reply:
This means that the old code would raise warnings right?

For example look at this log https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-linux-trusty-py3.5-test/18969/consoleFull